file_name: "teleop_recording_2024-02-22-10-53-08"
segmentation_indices: [ 3200,  6200,  8700, 12850, 15350] # Optional
